pkg/k8shandler/defaults.go

```diff
@@ -19,4 +19,7 @@ var (
 
 	defaultFluentdMemory     resource.Quantity = resource.MustParse("736Mi")
 	defaultFluentdCpuRequest resource.Quantity = resource.MustParse("100m")
+	
+	defaultLogFileMetricExporterMemory     resource.Quantity = resource.MustParse("736Mi")
```

Comment:
this is too much for this go based container and likely does not require any defaults.  I would remove and only apply what a user specs if needed